shpy

```diff
@@ -113,6 +113,10 @@ getArgsForCall() {
         arg_file="$path/$i"
         if [ -f "$arg_file" ]; then
             args=$(cat "$arg_file")
+            if printf "%s" "$args" | grep -q " "; then
```

Comment:
Would `grep -qe "\s"` be more appropriate, for all whitespace?

Reply:
Good point.  I didn't think about an internal tab or other whitespace.  I added a parameter in the appropriate test.